### PR TITLE
print three times

### PR DESCRIPTION
**Pull request by Moksha36**
print three times and merge

### CODE REVIEW

File: lab6.cpp [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 int main()
```
Comment (Charlie37378) [inline lab6.cpp]: Just a wonderful job!

File: lab6.cpp [hunk 2]
```diff
@@ -2,6 +2,8 @@
...
 {
+  std::cout << "Hello, World!\n";
+  std::cout << "Hello, World!\n";
   std::cout << "Hello, World!\n";
```
Comment (Charlie37378) [inline lab6.cpp]: Love it!!

Comment (Moksha36) [inline lab6.cpp]: add two more ! after World here.

File: lab6.cpp [hunk 3]
```diff
@@ -4,6 +4,6 @@ int main()
 {
   std::cout << "Hello, World!\n";
   std::cout << "Hello, World!\n";
-  std::cout << "Hello, World!\n";
+  std::cout << "Hello, World!!!\n";
```
Comment (Moksha36) [inline lab6.cpp:7]: Resolved by commit c4a252ef089f21f410e20bcd72596c1ae3cbeda2

**Pull request closed without merging**